### ISSUE TITLE
Windows 11 does not evaluate the current working directory $EASYRSA correctly

### ISSUE THREAD

**Issue by TinCanTech** [OpenVPN/easy-rsa#467]
When using Windows 11, the use of `readlink` here:
https://github.com/OpenVPN/easy-rsa/blob/243bb708b02897c563631ce62633d90ae80c82ff/easyrsa3/easyrsa#L1669
`readlink` succeeds and returns a value such as `/cygdrive/c/PROGRA~1/OpenVPN/easy-rsa`

Under Windows, this `readlink` is expected to fail and not over-write the previous value for `prog_file`.

As a work-around, Windows 11 users can safely comment out line 1669.

**Comment by TinCanTech** [comment]
I am happy to help test with anybody using Windows 11 and Easy-RSA3.

**Comment by TinCanTech** [comment]
```
│19:25:43      wiscii | try this (it probably will not work):                                                          │
│19:25:58      wiscii | readlink -f "$0.exe"                                                                           │
│19:26:55  valorisa34 | EasyRSA Shell                                                                                  │
│19:26:55  valorisa34 | # readlink -f "$0.exe"                                                                         │
│19:27:34      wiscii | no output ?                                                                                    │
│19:27:52  valorisa34 | Deconnected                                                                                    │
│19:28:15      wiscii | you mean easyrsa closed ?                                                                      │
│19:28:47  valorisa34 | I have to refresh the page                                                                     │
│19:28:55         <-- | valorisa34 (~valorisa3@2a01cb0504948200288182c197448820.ipv6.abo.wanadoo.fr) has left          │
│                     | #tincantech                                                                                    │
│19:29:05      wiscii | oh :)                                                                                          │
│19:29:21         --> | valorisa34 (~valorisa3@2a01cb0504948200288182c197448820.ipv6.abo.wanadoo.fr) has joined        │
│                     | #tincantech                                                                                    │
│19:29:53  valorisa34 | Could you please give me the last command ?                                                    │
│19:30:05  valorisa34 | EasyRSA Shell                                                                                  │
│19:30:05  valorisa34 | # readlink -f "$0.exe"                                                                         │
│19:30:20      wiscii | yes                                                                                            │
│19:30:25      wiscii | that is correct                                                                                │
│19:30:40  valorisa34 | " /cygdrive/c/EasyRSA3/bin/sh.exe "                                                            │
│19:30:49      wiscii | ah-ha!!!                                                                                       │
│19:31:02      wiscii | we have the culprit !                                                                          │
```

**Comment by TinCanTech** [comment]
Closed via #528  unless somet'ing
